lib/cc/cli/analyze.rb

```diff
@@ -86,6 +88,20 @@ def path
         ENV['CODE_PATH']
       end
 
+      def registry_entry(engine_name)
+        if @dev_mode
+          make_dev_registry_entry(engine_name)
+        else
+          engine_registry[engine_name]
+        end
+      end
+
+      def make_dev_registry_entry(engine_name)
+        {
+          "image_name"=>"codeclimate/codeclimate-#{engine_name}:latest"
```

Comment:
Will it always be `codeclimate/`? I thought the `codeclimate-` prefix was so that someone could have a repo like `user/codeclimate-foo`